components/Cards/BUILD

```diff
@@ -182,21 +182,9 @@ mdc_snapshot_objc_library(
     ],
 )
 
-swift_library(
```

Comment:
This looks unrelated - is this just a clean up for empty sources?

Reply:
I got rid of the Cards Swift source dummy file so I had to make this change otherwise CI wouldn't pass